docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,252 @@
+                info = new CultureInfo(netLanguage);
+            }
+            catch (CultureNotFoundException e1)
+            {
+                Console.WriteLine("cannot find the current cultureInfo. so use 'en'. (" + e1.Message + ")");
+                info = new CultureInfo("en");
+            }
+
+            return info;
+        }
+
+        private void LanguageChanged(object sender, LocaleLanguageChangedEventArgs e)
+        {
+            CultureInfo info = GetCurrentCultureInfo();
+            _currentCultureInfo = info;
+            // Notify the change of locale information
+            MessagingCenter.Send<ILocalize, CultureInfo>(this, "LanguageChanged", info);
+        }
+
+        string TizenToDotnetLanguage(string tizenLanguage)
+        {
+            var netLanguage = tizenLanguage;
+            //certain languages need to be converted to CultureInfo equivalent
+            switch (tizenLanguage)
+            {
+                case "zh-CN":   // Chinese Simplified (People's Republic of China)
+                    netLanguage = "zh-Hans"; // correct code for .NET
+                    break;
+                case "zh-HK":  // Chinese Traditional (Hong Kong)
+                case "zh-hk":
+                case "zh-tw":  // Chinese Traditional (Taiwan)
+                case "zh-TW":
+                    netLanguage = "zh-Hant"; // correct code for .NET
+                    break;
+            }
+
+            Console.WriteLine("[Speedmeter] .NET Language/Locale:" + netLanguage);
+            return netLanguage;
+        }
+    }
+}
+
+```
+
+3. Use the DependencyService in the Xamarin.Forms application to call the interface and set your `RESX` resource culture to the correct value.
+You can receive the culture information using `MessagingCenter`. You can also update your application UI when your application is running and system language has been changed.
+
+```cs
+var ci = DependencyService.Get<ILocalize>().CurrentCultureInfo;
+Resx.AppResources.Culture = ci; // set the RESX for resource localization
+DependencyService.Get<ILocalize>().SetLocale(ci);
+
+// Whenever language has been changed, CurrentCulture will be updated.
+MessagingCenter.Subscribe<ILocalize, CultureInfo>(this, "LanguageChanged", (obj, culture) =>
+{
+    Resx.AppResources.Culture = culture; 
+    DependencyService.Get<ILocalize>().SetLocale(culture);
+    //update your App UI
+});
+```
+
+## Image Localization
+
+Tizen project supports localized images(resources) using different resource directories.
+Tizen uses res.xml file to specify the information about the directory, which contains the localized resources(For example, Image, Sound, and so on).
+The res.xml file is automatically generated when you build your application.
+
+**To add the localized resource directories in your application in Visual Studio:**
+
+- In Solution Explorer, select Tizen project. Go to **Tools &gt; Tizen &gt; Resource Manager**
+
+![local_res_mgr_1](media/local_res_mgr_1.png)
```

Comment:
Global comment : To make an image as a sub item of a list, 3 spaces are needed before the image like below.

```
- In Solution Explorer, select Tizen project. 

   ![local_res_mgr_1](media/local_res_mgr_1.png)
```

Reply:
done